.github/workflows/kind.yml

```diff
@@ -0,0 +1,26 @@
+name: Kind e2e Tests
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run e2e testing script
+        env:
+          GIT_API_KEY: ${{ secrets.GIT_API_KEY }}
```

Comment:
@moshloop we need to add the GIT_API_KEY to the repo secrets for GH Actions

Reply:
I'm keen to make the PR Comments a bit less noisy using [summaries](https://github.com/flanksource/build-tools/issues/9#issuecomment-634554506) and to make the reporting a bit more [obvious](https://github.com/flanksource/build-tools/pull/8) in terms of status, when the E2E vSphere testing is done.